### DISCUSSION TITLE
Couldn't integrate the skywalking agent in django application.

### DISCUSSION THREAD

**Discussion by me-srini** [apache/skywalking#11635] [Q&A]
I created a simple django project with a sample app.  While referring to the offl docs and integrating the skywalking tool with the application I faced some issues. The application is running fine but the metrics are not being pushed to the OAP-Server. It also states that the skywalking module is not found, though we installed skywalking package in pypi using "pip install apache-skywalking".

![Screenshot from 2023-12-07 13-01-23](https://github.com/apache/skywalking/assets/153169380/71e1e585-7682-45e5-93b8-27b3a83461da)

the documentation suggests the following code 
'''
from skywalking import agent, config
config.init(SomeConfig)
agent.start()
'''
They also displayed some parameters that can be passed using env variables. But when attempting, it was unsuccessful. It indicates the below dialogue.
![Screenshot from 2023-12-07 13-09-28](https://github.com/apache/skywalking/assets/153169380/124728c6-54d7-46aa-be59-be462509c494)

It would be helpful if anyone with the knowledge of this process provide us with a solution.

**Comment by wu-sheng** [comment]
Friendly ping @Superskyyy
